backend/src/Data/DataAccessRights.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore;
+
+namespace Database.Data;
+
+/// <summary>
+/// Access rights per dataset
+/// </summary>
+[Owned]
+public sealed class DataAccessRights
```

Comment:
As far as I see it, right now, I cannot model for example that all users and quantities are allowed but only certain institutions. To model those scenarios, I would make the dictionaries and collections below nullable (just by adding a `?` after the type declaration) with the meaning that `null` means that everything is allowed (nothing forbidden) for the respective dictionary/list.